### DISCUSSION TITLE
Cref的使用

### DISCUSSION THREAD

**Discussion by Chenyanlu99** [TheNetAdmin/zjuthesis#421] [General]
我想要将 \cref 命令生成的引用名称改为中文，例如将 “figure” 改为 “图”，但是使用以下代码报错了。应该如何修改？
```
\usepackage{cleveref}
\crefname{figure}{图}{图}
\Crefname{figure}{图}{图}
\crefname{table}{表}{表}
\Crefname{table}{表}{表}
\crefname{equation}{式}{式}
\Crefname{equation}{式}{式}
```

我是将以上代码放在了zjuthesis.tex文档的\documentclass[]前，但是无法运行。

**Comment by TheNetAdmin** [comment]
https://github.com/TheNetAdmin/zjuthesis/blob/master/config/format/general/reference.tex

建议使用hyperref管理引用的名称，一般`\autoref`即可

**Reply by Chenyanlu99** [reply]
嗯嗯，我之前不知道模版已经设置了`\autoref`，这样确实很方便~

**Comment by zhangbb-john** [comment]
各位好，我在使用 LaTeX 的 autoref 命令时遇到一个格式问题：

当前 autoref 生成的引用格式为“式 2.5”（中间为空格）

但我需要符合中文排版习惯的带括号格式，例如“式(2.5)”

我已尝试如下配置：

latex
\def\equationautorefname{式}
% 但输出仍为“式 2.5”，而非目标“式(2.5)”
问题：

这是 autoref 的功能限制吗？是否有办法通过配置实现括号的自动添加？

如果 autoref 无法实现，是否有推荐的其他宏包（如 cleveref）或自定义命令能优雅地解决此需求？

期待各位的经验分享，谢谢！
